src/solvers/sat/satcheck_minisat2.h

```diff
@@ -51,6 +53,9 @@ class satcheck_minisat2_baset:public cnf_solvert
     time_limit_seconds=lim;
   }
 
+  void set_random_var_freq(std::size_t) override;
```

Comment:
Do the same for glucose to see whether this can be used with at least two solvers without hiccup.